tests/FakeItEasy.Tests/Configuration/RuleBuilderTests.cs

```diff
@@ -52,7 +52,7 @@ public void Returns_called_with_value_returns_parent_configuration()
 
             var result = returnConfig.Returns(10);
 
-            result.Should().Be(this.builder);
+            result.Should().BeSameAs(returnConfig);

```

Comment:
Oh, yeah. I'd've convinced myself that I had good specs in place to cover whatever we're trying to cover here, and dropped these like hot potatoes.